src/lib/core/CHIPConfig.h

```diff
@@ -1845,6 +1845,17 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #define CHIP_CONFIG_MAX_BDX_LOG_TRANSFERS 5
 #endif // CHIP_CONFIG_MAX_BDX_LOG_TRANSFERS
 
+/**
+ *  @def CHIP_CONFIG_TEST_GOOGLETEST
+ *
+ *  @brief
+ *    If asserted (1), enable APIs that support unit tests built with the GoogleTest framework
+ *
+ */
+#ifndef CHIP_CONFIG_TEST_GOOGLETEST
```

Comment:
What is this used for?

Reply:
It's a change on `origin/master`. I accidentally didn't remove it during the rebase / cherry-pick when I updated the PR. You shouldn't see it in the latest rev.